src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -204,6 +204,11 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
   const onBarCodeScanned: BarCodeScannedCallback = (event) => {
     if (isFocused && isScanningEnabled && event.data) {
       onCheck(event.data);
+      if (event.type.includes("Code39")) {
```

Comment:
May be better to capitalise this and then perform a comparison, so even when the capitalisation in `type` changes, you are still able to compare them.

Reply:
this one is because `event` looks like this
```
Object {
  "data": "S0000001I",
  "target": 689,
  "type": "org.iso.Code39",
}
```
should I still change it though? wdyt 🤔

Reply:
How about this?

```suggestion
      if (event.type === BarCodeScanner.Constants.BarCodeType.code39)) {
```

Alternatively, do you think we should:

- Pass `passportId` if present in `event.data`
- Pass `event.data` otherwise

Instead of changing what we check based on the scanner type.

Reply:
i think we should pass `passportId`, cus as of the story, QR for passport is an `object` while the Barcode for NRIC is a `string`, reason being we want to use `object` so that we can pass other data if needed.
With that said, I think can use the provided constant instead as you suggested!